source/administration/config-settings.rst

```diff
@@ -3533,6 +3544,17 @@ Enable Tutorial (Experimental)
 | This feature’s ``config.json`` setting is ``"EnableTutorial": true`` with options ``true`` and ``false`` for above settings respectively.  | 
 +--------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable Join Leave Messages in Default Channel (Experimental)
```

Comment:
Is this different from this: https://docs.mattermost.com/administration/config-settings.html?highlight=config%20settings#enable-default-channel-leave-join-system-messages?